d8/d8batch/modules/d8feedsourcect/config/install/field.field.node.feed_source.field_feed_source_url.yml

```diff
@@ -0,0 +1,25 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_feed_source_url
+    - node.type.feed_source
+  module:
+    - link
+  enforced:
+    module:
+      - d8feedsourcect
+id: node.feed_source.field_feed_source_url
+field_name: field_feed_source_url
+entity_type: node
+bundle: feed_source
+label: URL
+description: ''
+required: true
+translatable: false
+default_value: {  }
+default_value_callback: ''
+settings:
+  link_type: 16
```

Comment:
I know it's not your fault but core's, but what does 16 mean in this context? :)